Modules/overlapped.c

```diff
@@ -612,6 +639,31 @@ Overlapped_dealloc(OverlappedObject *self)
     SetLastError(olderr);
 }
 
+static PyObject*
+unparse_address(LPSOCKADDR Address, DWORD Length) {
```

Comment:
The mix of CamelCase and lower_case looks a little inaccurate.
Would you convert variables to lower case?

Reply:
Nevermind, the file doesn't follow any single casing schema.